extensions/positron-python/src/client/positron/manager.ts

```diff
@@ -101,8 +102,41 @@ export class PythonRuntimeManager implements IPythonRuntimeManager {
      * Recommend a Python language runtime based on the workspace.
      */
     async recommendedWorkspaceRuntime(): Promise<positron.LanguageRuntimeMetadata | undefined> {
-        // TODO: This is where we could recommend a runtime based on the
-        // workspace, e.g. if it contains a virtualenv
+        // TODO: may need other handling for multiroot workspaces
```

Comment:
Might be a little tidier to extract a `recommendedWorkspaceInterpreterPath` method here that returns an object with the path and immediate flag, since you can use early exits in that method.